site/src/pages/TaskPage/TaskStartupAlert.tsx

```diff
@@ -0,0 +1,101 @@
+import Link from "@mui/material/Link";
+import type { WorkspaceAgent } from "api/typesGenerated";
+import { Alert } from "components/Alert/Alert";
+import type { FC } from "react";
+import { docs } from "utils/docs";
+
+type TaskStartupAlertProps = {
+	agent: WorkspaceAgent;
+};
+
+export const TaskStartupAlert: FC<TaskStartupAlertProps> = ({ agent }) => {
+	const lifecycleState = agent.lifecycle_state;
+
+	if (lifecycleState === "start_error") {
+		return <ErrorScriptAlert />;
+	}
+
+	if (lifecycleState === "start_timeout") {
+		return <TimeoutScriptAlert />;
+	}
+
+	return null;
+};
+
+const ErrorScriptAlert: FC = () => {
+	return (
+		<Alert severity="warning" dismissible>
+			The workspace{" "}
+			<Link
+				title="startup script has exited with an error"
```

Comment:
```suggestion
				title="Startup script has exited with an error"
```
Titles across the repo seem to start with upper case.